src/Analyzer/FunctionSecretArgumentsFinderTreeNode.h

```diff
@@ -3,31 +3,48 @@
 #include <Parsers/FunctionSecretArgumentsFinder.h>
 #include <Analyzer/ConstantNode.h>
 #include <Analyzer/FunctionNode.h>
+#include <Analyzer/TableFunctionNode.h>
 #include <Analyzer/IdentifierNode.h>
 
 
-namespace DB
+namespace
+{
```

Comment:
we don't need these templates in global namespace, we only need instantiations `FunctionSecretArgumentsFinderTreeNode` and `TableFunctionSecretArgumentsFinderTreeNode`
and I made functions inline to prevent bloating... I guess...

Reply:
well, anyway clang tidy doesn't allow anonymous namespace in a header :)
will fix